daemon/metrics_unix.go

```diff
@@ -34,52 +34,22 @@ func (daemon *Daemon) listenMetricsSock() (string, error) {
 	return path, nil
 }
 
-func registerMetricsPluginCallback(getter plugingetter.PluginGetter, sockPath string) {
-	getter.Handle(metricsPluginType, func(name string, client *plugins.Client) {
+func registerMetricsPluginCallback(store *plugin.Store, sockPath string) {
+	store.RegisterRuntimeOpt(metricsPluginType, func(s *specs.Spec) {
+		f := plugin.WithSpecMounts([]specs.Mount{
```

Comment:
Please update https://github.com/docker/cli/blob/master/docs/extend/config.md with this new option.

Reply:
I don't think there's a new option here?

Reply:
Sorry, not a user facing option. More of a methodology of how a plugin author can customize. But thinking more, its not necessary atm.